boards/arm/stm32f407g_disc1/doc/stm32f407g_disc1.rst

```diff
@@ -0,0 +1,202 @@
+
+- Eight LEDs:
+
+       - USB communication (LD1)
+       - 3.3 V power on (LD2)
+       - Four user LEDs: orange (LD3), green (LD4), red (LD5), and blue (LD6)
+       - 2 USB OTG LEDs for VBUS (LD7) and over-current (LD8)
+
+- Two push-buttons: USER and RESET
+- USB OTG FS with micro-AB connector
+- LIS302DL or LIS3DSH ST MEMS 3-axis accelerometer
+- MP45DT02 ST-MEMS audio sensor omni-directional digital microphone
+- CS43L22 audio DAC with integrated class D speaker driver
+
+.. image:: img/en.stm32f4_discovery.jpg
+     :width: 350px
+     :align: center
+     :height: 550px
+     :alt: STM32F407G-DISC1
+
+More information about the board can be found at the `STM32F407G-DISC1 website`_.
+
+Hardware
+********
+
+STM32F407G-DISC1 Discovery kit provides the following hardware components:
+
+- STM32F407VGT6 in LQFP100 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 168 MHz max CPU frequency
+- VDD from 1.8 V to 3.6 V
+- 1 MB Flash
+- 192+4 KB SRAM including 64-Kbyte of core coupled memory
+- GPIO with external interrupt capability
+- 3x12-bit ADC with 24 channels
+- 2x12-bit D/A converters
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (17)
+- Watchdog Timers (2)
+- USART/UART (6)
+- I2C (3)
+- SPI (3)
+- SDIO
+- 2xCAN
+- USB 2.0 OTG FS with on-chip PHY
+- USB 2.0 OTG HS/FS with dedicated DMA, on-chip full-speed PHY and ULPI
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F407VG can be found here:
+       - `STM32F407VG on www.st.com`_
+       - `STM32F407 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f407g_disc1 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
```

Comment:
delete the blank line and the leading spaces before  the filename

    The default configuration can be found in the defconfig file:
    ``boards/arm/stm32f407g_disc1/stm32f407g_disc1_defconfig``